fdb-record-layer-core/src/test/proto/evolution/test_field_type_change.proto

```diff
@@ -24,7 +24,7 @@ package com.apple.foundationdb.record.evolution.fieldtypechange;
 option java_package = "com.apple.foundationdb.record.evolution";
 option java_outer_classname = "TestFieldTypeChangeProto";
 
-import "record_metadata_options.proto";
+//import "record_metadata_options.proto";
```

Comment:
Why? What was broken?

Reply:
Not sure what happened. I changed something somewhere in a test proto file and suddenly got all these warnings about unnecessary imports. And since we run everything in `-Werror`...